docs/DEVGUIDE.md

```diff
@@ -1,10 +1,15 @@
-# An End-to-End Walkthrough
+# Developer's Guide to Service-Catalog
 
-## Prerequisites and Setup
-
-You will need to meet the following prerequisites:
+## Prerequisites
 
+At a minimum you will need:
 * [Docker](https://www.docker.com) installed locally
+* GNU Make
+* [git](https://git-scm.com)
+
+These will allow you to build and test using Docker.
+
+If you want to build without Docker, then you will also need:
```

Comment:
I think that makes a lot of sense.  However, would you be ok if I did that in a follow-on PR?  The reason I'm hesitant to do it here is because I think its going to require a lot of work to get it right. For example, we're going to have to decide what is the best mechanism to deploy k8s in Docker (e.g. hyperkube?).  Might not be very controversial but I'd like to get the current state of things clean-up first because I add too much new stuff.  I can open issue to remind myself to do it - if you're ok with it.